src/User.cpp

```diff
@@ -877,11 +877,11 @@ const CString& CUser::GetTimestampFormat() const { return m_sTimestampFormat; }
 bool CUser::GetTimestampAppend() const { return m_bAppendTimestamp; }
 bool CUser::GetTimestampPrepend() const { return m_bPrependTimestamp; }
 
-bool CUser::IsValidUserName(const CString& sUserName) {
+bool CUser::IsValidUserName(const CString& sUsername) {
```

Comment:
Function names too, but need to also keep old names for now, to not break existing modules; they should be marked as deprecated though.

Reply:
OK, will fix.